locales/en.json

```
@@ -115,7 +115,7 @@
     "backup_output_directory_forbidden": "Pick a different output directory. Backups can not be created in /bin, /boot, /dev, /etc, /lib, /root, /run, /sbin, /sys, /usr, /var or /home/yunohost.backup/archives sub-folders",
     "backup_output_directory_not_empty": "You should pick an empty output directory",
     "backup_output_directory_required": "You must provide an output directory for the backup",
-    "backup_output_symlink_dir_broken": "You have a broken symlink in place of your archive directory '{path:s}'. You may have a specific setup to backup your data on another filesystem, in this case you probably forgot to remount or plug in your hard-drive or USB key.",
+    "backup_output_symlink_dir_broken": "Your archive directory '{path:s}' only contains a broken symlink. Maybe you forgot to re/mount or plug in the storage medium it points to.",
```

Imho it's more about the archive directory itself being a symlink rather than containing a symlink?

```suggestion
    "backup_output_symlink_dir_broken": "Your archive directory '{path:s}' points to a broken symlink. Maybe you forgot to re/mount or plug in the storage medium it points to.",
```